.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+name: Feature Request
+description: New features and enhancement ideas for the Upbound product.
+labels: enhancement
+title: "<component name>: <feature overview>"
```

Comment:
Do we need the <component name> in the title since we have the product-area dropdown? We could automatically label issues based on the value selected.

Reply:
Doing that automatically would be awesome!